sky/clouds/service_catalog/lambda_catalog.py

```diff
@@ -62,13 +62,7 @@ def get_vcpus_from_instance_type(instance_type: str) -> Optional[float]:
 def get_default_instance_type(cpus: Optional[str] = None) -> Optional[str]:
     if cpus is None:
         cpus = str(_DEFAULT_NUM_VCPUS)
-    # Set to gpu_1x_a100_sxm4 to be the default instance type if match vCPU
-    # requirement.
-    df = _df[_df['InstanceType'].eq('gpu_1x_a100_sxm4')]
-    instance = common.get_instance_type_for_cpus_impl(df, cpus)
-    if not instance:
-        instance = common.get_instance_type_for_cpus_impl(_df, cpus)
-    return instance
+    return common.get_instance_type_for_cpus_impl(_df, cpus)
```

Comment:
I changed it because was cheaper to test when developing multi-node. I can revert back (from my experience, the original default instance is available is more regions).

Reply:
Either default instance sounds good to me, then. : )